app/views/passwordless/sessions/new.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_for @session, url: send(authenticatable_resource).sign_in_path do |f| %>
+<%= form_for @session, url: send(Passwordless.mounted_as).sign_in_path do |f| %>
```

Comment:
Does this mean, we can remove that helper method?

Reply:
Good catch. Removed it!